src/index.js

```diff
@@ -264,6 +264,13 @@ Observable.prototype.bufferWhile = function(fn, options) {
   return bufferWhile(this, fn, options);
 };
 
+// (Stream, number) -> Stream
+// (Property, number) -> Property
+const bufferWithCount = require('./one-source/buffer-with-count');
+Observable.prototype.bufferWithCount = function(count) {
```

Comment:
You don't pass `options` here, that is why `{flushOnEnd: false}` doesn't work probably.